src/erlcloud_aws.erl

```diff
@@ -187,16 +189,17 @@ aws_request4_no_update(Method, Protocol, Host, Port, Path, Params, Service,
                     Region, Service, [])
     end,
     aws_request_form(Method, Protocol, Host, Port, Path, Query,
-                     SignedHeaders ++ Headers, Config).
+                     SignedHeaders ++ RequestHeaders, Config).
 
 
 -spec aws_request_form(Method :: atom(), Protocol :: undefined | string(), Host :: string(),
                         Port :: undefined | integer() | string(), Path :: string(), Form :: string(),
                         Headers :: list(), Config :: aws_config()) -> {ok, binary()} | {error, tuple()}.
 aws_request_form(Method, Protocol, Host, Port, Path, Form, Headers, Config) ->
-    RequestHeaders = [{"content-type",
-                      "application/x-www-form-urlencoded; charset=utf-8"} |
-                     Headers],
+    RequestHeaders = case proplists:is_defined("content-type", Headers) of
```

Comment:
last question for this PR
seems this case is unnecessary given the cahgnes at line 176 above where the `"content-type"` is always added.
agree?

Reply:
`aws_request_form` method is also used [here](https://github.com/erlcloud/erlcloud/blob/81243b191550256c6452e35f7455d89b15759bac/src/erlcloud_ses.erl#L989)